src/Button/button-group.mdx

```diff
@@ -1,6 +1,9 @@
 ---
 title: 'Button Group'
 type: 'component'
```

Comment:
[clarification] Is this button-group.mdx documentation actually reachable by the technical documentation site navigation? There's obviously a page for the `Button` but I'm not seeing this page for ButtonGroup/ButtonToolbar in the navigation. Do you think it should be?

Reply:
You could reach it in `All components (A-Z)` section at the bottom of the documentation (that's how I found it actually, I was going in alphabetical order to update the components), but it didn't belong to any other category which is weird. I've added it to `Buttonlike` category for now, I believe it definitely belongs there.

I'm wondering if we need a separate page for ButtonToolbar component, seems like it's closely connected to the ButtonGroup and one page should be enough. Do you think we should add another page for it?

Reply:
I agree, one page should be enough here.